pkg/commands/git_commands/stash_loader.go

```diff
@@ -75,8 +76,25 @@ func (self *StashLoader) getUnfilteredStashEntries() []*models.StashEntry {
 }
 
 func (c *StashLoader) stashEntryFromLine(line string, index int) *models.StashEntry {
-	return &models.StashEntry{
+	model := &models.StashEntry{
 		Name:  line,
 		Index: index,
 	}
+
+	regular := regexp.MustCompile(`(?P<line>.*?) date@\{(?P<age>.*?)\}`)
+	match := regular.FindStringSubmatch(line)
```

Comment:
OK, my bad since I advised you to go with the code from the issue comment without looking at it closely enough.

I think this can be simplified so that we don't need this regex business. If you make the format string `"--pretty=%ct|%gs"`, then you can simply split the string on `|`. I picked `|` arbitrarily, any non-digit character will do.

Something like this:
```go
tstr, msg, ok := strings.Cut(line, "|")
if ok {
	model.Name = msg
	t, err := strconv.ParseInt(tstr, 10, 64)
	if err == nil {
		model.Recency = utils.UnixToTimeAgo(t)
	}
}
return model
```

Reply:
I was unsure about the regex mechanism, but I couldn't think anything better. Forcing the information to parse at the beginning of the string works great, instead! Thanks for the suggestion 🙏🏻